test/e2e/apps/rc.go

```diff
@@ -382,8 +382,7 @@ var _ = SIGDescribe("ReplicationController", func() {
 				}
 			}
 			if !foundRc {
-				framework.Logf("Got unexpected replication controller list %v", rcs.Items)
-				framework.Failf("could not find ReplicationController %s", testRcName)
+				framework.Failf("ReplicationController doesn't has expected values.")
```

Comment:
Also dump the list here.

Reply:
Dump it inside the failure message... :sweat_smile:

Reply:
Oh... Ok 😄